ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -915,7 +915,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${PROJECT_DIR}/Crashlytics/Fabric.framework/run\" FABRIC_API_KEY_HERE FABRIC_SECRET_HERE";
+			shellScript = "if [ \"$releaseConfig\" = \"Release\" ]; then\n    \"${PROJECT_DIR}/Crashlytics/Fabric.framework/run\" FABRIC_API_KEY_HERE FABRIC_SECRET_HERE\nfi";
```

Comment:
Can we make this check some other configuration option? I often run the app in Release config for testing and this will break that workflow